src/main/java/net/snowflake/client/core/JsonSQLOutput.java

```diff
@@ -0,0 +1,115 @@
+package net.snowflake.client.core;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.InputStream;
+import java.io.Reader;
+import java.math.BigDecimal;
+import java.net.URL;
+import java.sql.*;
+import java.sql.Date;
+import java.util.*;
+
+public class JsonSQLOutput implements SQLOutput {
+
+  private Vector attribs = new Vector();
+  private Map map = new HashMap();
+  private final ObjectMapper OBJECT_MAPPER = ObjectMapperFactory.getObjectMapper();
+
+  public String getJsonString() {
+    try {
+      return OBJECT_MAPPER.writeValueAsString(attribs);
+    } catch (JsonProcessingException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  @Override
+  public void writeString(String value) throws SQLException {
+    attribs.add(value);
+  }
+
+  @Override
+  public void writeBoolean(boolean x) throws SQLException {}
```

Comment:
what about ale the methods with empty body? are we going to fill them and there should be TODOs comments? or should we add throwing exception there?